authorization-service/src/functions/basicAuthorizer/index.ts

```diff
@@ -0,0 +1,27 @@
+import schema from './schema';
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
```

Comment:
It seems to me we don't need to assign any event for authorizer lambda. It's called by another methods which require authorization